new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -1337,7 +1468,14 @@ private static void createDomainCrAndVerify(String adminSecretName,
             .serverPod(new ServerPod()
                 .addEnvItem(new V1EnvVar()
                     .name("JAVA_OPTIONS")
-                    .value("-Dweblogic.StdoutDebugEnabled=false"))
+                    .value("-Dweblogic.StdoutDebugEnabled=false "
+                        + "-Dweblogic.security.SSL.ignoreHostnameVerification=true -Dssl.debug=true "
+                        + "-Djavax.net.debug=ssl:handshake "
+                        //+ "-DUseSunHttpHandler=true -Dssl.debug=true "
```

Comment:
Remove the commented code.  Do we need to set the SSL debug ?  javax.net.debug=all generate a tons of message in server logs

Reply:
will remove, it was done for debugging purposes, now can remove